Backpack-SwiftUI/Skeleton/Classes/BPKShimmer.swift

```diff
@@ -19,49 +19,34 @@
 import SwiftUI
 
 struct BPKShimmer: ViewModifier {
-    @State private var phase: CGFloat = 0
+    @State private var offset = 0.0
     private let duration = 1.0
     private let bounce = false
-    private let delay = 0.2
```

Comment:
The dealy property was never used before

Reply:
👍